jpo-ode-plugins/src/test/java/us/dot/its/jpo/ode/plugin/j2735/J2735BsmPart2ContentTest.java

```diff
@@ -1,52 +1,50 @@
 package us.dot.its.jpo.ode.plugin.j2735;
 
-import org.junit.jupiter.api.Test;
-
-import static org.junit.Assert.*;
-
-
-import static us.dot.its.jpo.ode.plugin.j2735.J2735BsmPart2Content.J2735BsmPart2Id.*;
+import static us.dot.its.jpo.ode.plugin.j2735.J2735BsmPart2Content.J2735BsmPart2Id.SpecialVehicleExtensions;
+import static us.dot.its.jpo.ode.plugin.j2735.J2735BsmPart2Content.J2735BsmPart2Id.SupplementalVehicleExtensions;
+import static us.dot.its.jpo.ode.plugin.j2735.J2735BsmPart2Content.J2735BsmPart2Id.VehicleSafetyExtensions;
 
+import org.junit.jupiter.api.Assertions;
+import org.junit.jupiter.api.Test;
 import us.dot.its.jpo.ode.util.JsonUtils;
 
 public class J2735BsmPart2ContentTest {
 
-
-    
     @Test
     public void shouldDeserializeBsmPart2Content_VehicleSafetyExtensions() {
 
         final String bsmJson = "{\"id\":\"VehicleSafetyExtensions\",\"value\":{\"pathHistory\":{\"crumbData\":[{\"elevationOffset\":0.0,\"latOffset\":-0.0000491,\"lonOffset\":-0.0000043,\"timeOffset\":0.39},{\"elevationOffset\":-1.1,\"latOffset\":-0.0007303,\"lonOffset\":-0.0001015,\"timeOffset\":6.79},{\"elevationOffset\":-1.1,\"latOffset\":-0.0012664,\"lonOffset\":-0.0002581,\"timeOffset\":10.9},{\"elevationOffset\":-0.7,\"latOffset\":-0.0018413,\"lonOffset\":-0.0005267,\"timeOffset\":14.39}]},\"pathPrediction\":{\"confidence\":70.0,\"radiusOfCurve\":-3139.0}}}";
 
         var deserialized = (J2735BsmPart2Content)JsonUtils.fromJson(bsmJson, J2735BsmPart2Content.class);
 
-        assertNotNull(deserialized);
-        assertEquals(deserialized.getId(), VehicleSafetyExtensions);
-        assertTrue(deserialized.getValue() instanceof J2735VehicleSafetyExtensions);
+        Assertions.assertNotNull(deserialized);
+        Assertions.assertEquals(VehicleSafetyExtensions, deserialized.getId());
+      Assertions.assertInstanceOf(J2735VehicleSafetyExtensions.class, deserialized.getValue());
         var extensions = (J2735VehicleSafetyExtensions)deserialized.getValue();
-        assertNotNull(extensions.getPathHistory());
-        assertNotNull(extensions.getPathPrediction());
+        Assertions.assertNotNull(extensions.getPathHistory());
+        Assertions.assertNotNull(extensions.getPathPrediction());
     }
 
     @Test
     public void shouldDeserializeBsmPart2Content_SupplementalVehicleExtensions() {
-        final String bsmJson = "{\"id\":\"SupplementalVehicleExtensions\",\"value\":{\"classification\":null,\"classDetails\":null,\"vehicleData\":null,\"weatherReport\":null,\"weatherProbe\":null,\"obstacle\":null,\"status\":null,\"speedProfile\":null,\"theRTCM\":null}}";
+        final String bsmJson = "{\"id\":\"SupplementalVehicleExtensions\",\"value\":{\"classification\":null,\"classDetails\":null,\"vehicleData\":null,\"doNotUse1\":null,\"doNotUse2\":null,\"doNotUse3\":null,\"status\":null,\"doNotUse4\":null,\"doNotUse5\":null}}";
```

Comment:
We should separate the JSON in this file into individual JSON files and make sure they include all possible optional values from the schema.

Reply:
Addressed